packages/ui/src/composites/wui-transaction-list-item-loader/index.ts

```diff
@@ -0,0 +1,31 @@
+import { html, LitElement } from 'lit'
+import '../../components/wui-text/index.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import '../wui-transaction-visual/index.js'
+import styles from './styles.js'
+
+@customElement('wui-transaction-list-item-loader')
+export class WuiTransactionListItemLoader extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
```

Comment:
I don't think you need all of these styles here i.e. element ones are when we use things like buttons / forms etc. but just in case double verify 😅 been a while since I touched this area.